src/styles/stopwatch/stopwatch.scss

```diff
@@ -0,0 +1,41 @@
+.main {
+  @include horizontal-center-flex;
+
+  min-height: 100vh;
+
+  margin: 0;
+}
```

Comment:
better to move this styles to individual .scss file to make your BEM block more independent